blog-service/2024-08-21-search.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Behaviour Change for `count` Operator (Search)
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - count
+  - search-operator
+  - log-search
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're happy to announce a change in the behavior of the `count` operator. Previously, the `count` operator was supported by the `Int` data type, which could result in a negative count value after when exceeding the maximum integer value. To address this, we are changing the data type of the `count` operator from `Int` to `Long` data type to support the handling of large numeric values. 
+
+This change will apply to all new queries. However, it may pose challenges for existing saved searches, scheduled searches, searches using the `count` operator as part of the scheduled view, and searches using the `count` operator as part of lookup tables. Below are some conditions to address these challenges:
+
+- For existing scheduled views, the query will use the `Int` data type, while new scheduled views will utilize the `Long` data type for the `count` operator.
```

Comment:
the scheduled view related callout should be added to the Scheduled Views page with a callout on the release note for users to expect newer scheduled views to support Long as the data type for count operator